source/include/drivers/cs_Gpio.h

```diff
@@ -32,13 +35,13 @@ class Gpio : public EventListener {
 	static Gpio& getInstance();
 
 	//! Initialize (from cs_Crownstone as driver)
-	void init(const boards_config_t & board);
+	void init(const boards_config_t* board);
```

Comment:
Any reason behind this change from passing by reference to passing a pointer?

Reply:
No reason, apart from the fact that I looked at the PowerSampling class as inspiration which also passes by pointer.

Reply:
That's on older class. However, more importantly, it makes sense to pass by reference if you expect that something can also be arguably `null`. If that doesn't make sense, just pass by reference.

In your case, it's even worse. You changed to a pointer and then didn't check if the pointer is null in the function body to begin with. See https://github.com/crownstone/bluenet/blob/543b6268e69899d83629df59a22dce9841e704cc/source/src/drivers/cs_Gpio.cpp#L29 

I'd prefer to have this reverted to pass by reference.